evalml/automl/utils.py

```diff
@@ -77,6 +80,11 @@ def make_data_splitter(
         sklearn.model_selection.BaseCrossValidator: Data splitting method.
     """
     random_seed = random_seed
+    if n_splits is None:
```

Comment:
@christopherbunn I think it's worth running perf tests before merge. What do you think? 

We may need to artificially shorten our datasets to meet the threshold but I'm curious by how much this slows us down and if it will make our small-dataset models better?

Reply:
For sure, I was planning on kicking off a set today. There's none of them that are below 100 rows, but there are a good chunk that are 500 rows or less (which is the threshold suggested in #1783). If we want to increase the threshold to 500, I should be able to assemble a set of small datasets easily but I might have to make a new artificial set for <100 rows.

Reply:
Sweet, sounds good! I don't have a strong opinion on what the threshold should be so happy to keep with `100` and artificially shorten some of the existing datasets to meet the threshold.

Reply:
I agree with freddy with respect to artificially shortening.  If you're going to do that, I'd suggest tying the max number of rows in the dataset to a variable so we can adjust that lever later on all the tests at once.